code/modules/antagonists/heretic/items/madness_mask.dm

```diff
@@ -0,0 +1,75 @@
+// The spooky "void" / "abyssal" / "madness" mask for heretics.
+/obj/item/clothing/mask/madness_mask
+	name = "abyssal mask"
+	desc = "A mask created from suffering. When you look into its eyes, it looks back."
+	icon_state = "mad_mask"
+	item_state = null
+	w_class = WEIGHT_CLASS_SMALL
+	flags_cover = MASKCOVERSEYES
+	resistance_flags = FLAMMABLE
+	flags_inv = HIDEEARS|HIDEEYES|HIDEFACE
+	flags_cover = MASKCOVERSMOUTH | MASKCOVERSEYES
+	///Who is wearing this
+	var/mob/living/carbon/human/local_user
+
+/obj/item/clothing/mask/madness_mask/Destroy()
+	local_user = null
+	return ..()
+
+/obj/item/clothing/mask/madness_mask/examine(mob/user)
+	. = ..()
+	if(IS_HERETIC_OR_MONSTER(user))
+		. += "<span class='notice'>Actively drains the sanity and stamina of nearby non-heretics when worn.</span>"
+		. += "<span class='notice'>If forced onto the face of a non-heretic, they will be unable to remove it willingly.</span>"
+	else
+		. += "<span class='danger'>The eyes fill you with dread... You best avoid it.</span>"
+
+/obj/item/clothing/mask/madness_mask/equipped(mob/user, slot)
+	. = ..()
+	if(!(slot & ITEM_SLOT_MASK))
+		return
+	if(!ishuman(user) || !user.mind)
+		return
+
+	local_user = user
+	START_PROCESSING(SSobj, src)
+
+	if(IS_HERETIC_OR_MONSTER(user))
+		return
+
+	flags |= NODROP
+	to_chat(user, "<span class='userdanger'>[src] clamps tightly to your face as you feel your soul draining away!</span>")
```

Comment:
woah this is just like lethal company

Reply:
Hey louis, this was just like the time I went to Rend with my pals in lethal company and I tried the mask on, eheheheh